packages/m2m-gateway/test/api/purposes/createPurposeVersion.test.ts

```diff
@@ -0,0 +1,104 @@
+import { describe, it, expect, vi } from "vitest";
+import { generateToken } from "pagopa-interop-commons-test";
+import { AuthRole, authRole } from "pagopa-interop-commons";
+import request from "supertest";
+import { m2mGatewayApi } from "pagopa-interop-api-clients";
+import { unsafeBrandId } from "pagopa-interop-models";
+import { api, mockPurposeService } from "../../vitest.api.setup.js";
+import { appBasePath } from "../../../src/config/appBasePath.js";
+import {
+  missingMetadata,
+  purposeVersionNotFound,
+  resourcePollingTimeout,
+} from "../../../src/model/errors.js";
+import {
+  getMockedApiPurpose,
+  getMockedApiPurposeVersion,
+} from "../../mockUtils.js";
+
+describe("POST /purposes/:purposeId/versions router test", () => {
+  const mockPurposeVersion = getMockedApiPurposeVersion();
+  const mockPurpose = getMockedApiPurpose({
+    versions: [mockPurposeVersion],
+  });
+
+  const mockPurposeVersionSeed: m2mGatewayApi.PurposeVersionSeed = {
+    dailyCalls: mockPurposeVersion.dailyCalls,
+  };
+
+  const makeRequest = async (
+    token: string,
+    purposeId: string,
```

Comment:
```suggestion
    purposeId: PurposeID,
```
Same for the other tests

Reply:
I think it's better to keep it as a string, in this case.

See [this](https://github.com/pagopa/interop-be-monorepo/pull/1870#discussion_r2092868719) explanation